app/src/main/java/ch/epfl/favo/favor/FavorDatabase.java

```diff
@@ -0,0 +1,33 @@
+package ch.epfl.favo.favor;
+
+import ch.epfl.favo.common.NotImplementedException;
+
+public class FavorDatabase {
```

Comment:
The singleton pattern here could be a good idea, be sure to handle it correctly however.